### PR TITLE
feat: Add support for `backdrop-filter` property in style panel

### PR DESCRIPTION
**Pull request by JayaKrishnaNamburu**
## Description

This PR adds support for using `backdrop-filter` property in the style-panel. Here are more details on the property to test it around.
https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter

## Steps for reproduction

1. Add an element to the canvas.
2. Click on the "+" icon in the style panel next to the `Backdrop Filters` section.
3. Edit the default filter that is added.
4. Swap the layers and publish the project.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [x] made a self-review
- [x] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [x] tested locally and on preview environment (preview dev login: 5de6)

### CODE REVIEW

**Review by kof — APPROVED** [review]
- [x] lets order it after filters
- [ ] noticed its not saving when closing/blur

File: apps/builder/app/builder/features/style-panel/sections/backdrop-filter/backdrop-filter.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+              label={label}
+              tooltip={
+                <Tooltip
+                  content={
+                    <Flex gap="2" direction="column">
+                      <Text variant="regularBold">{label}</Text>
+                      <Text variant="monoBold">backdrop-filter</Text>
+                      <Text>
+                        Applies graphical effects like
+                        <br />
```
Comment (kof) [inline apps/builder/app/builder/features/style-panel/sections/backdrop-filter/backdrop-filter.tsx:97]: did you need to add br?  wondering why

File: apps/builder/app/builder/features/style-panel/sections/filter/filter.tsx [hunk 2]
```diff
@@ -80,7 +82,28 @@ export const Section = (props: SectionProps) => {
...
+              label={label}
+              tooltip={
+                <Tooltip
+                  content={
+                    <Flex gap="2" direction="column">
+                      <Text variant="regularBold">{label}</Text>
+                      <Text variant="monoBold">filter</Text>
+                      <Text>
+                        Applies graphical effects like
+                        <br />
```
Comment (kof) [inline apps/builder/app/builder/features/style-panel/sections/filter/filter.tsx:97]: same question

File: apps/builder/app/builder/features/style-panel/sections/sections.ts [hunk 3]
```diff
@@ -35,6 +36,7 @@ export const sections = new Map<
   ["boxShadows", boxShadows],
   ["filter", filter],
   ["transitions", transitions],
+  ["backdropFilters", backdropFilter],
```
Comment (kof) [inline apps/builder/app/builder/features/style-panel/sections/sections.ts:39]: ordering

File: apps/builder/app/builder/features/style-panel/sections/filter/filter-content.tsx [hunk 4]
```diff
@@ -7,12 +7,9 @@ import {
   Flex,
```
Comment (kof) [inline apps/builder/app/builder/features/style-panel/sections/filter/filter-content.tsx:7]: btw need to use code editor in all of those code areas instad of textarea, can be a separate pr though

**Comment by kof** [comment]
- [ ] Unrelated to this specific pr, can be done separately: when there is only one layer, drag handle shouldn't be visible

**Comment by kof** [comment]
- [ ] delete backdrop-filter from advanced panel short list

File: apps/builder/app/builder/features/style-panel/sections/backdrop-filter/backdrop-filter.tsx (hunk 1 above)
Comment (JayaKrishnaNamburu) [reply apps/builder/app/builder/features/style-panel/sections/backdrop-filter/backdrop-filter.tsx:97]: Nope, Just took it out of the content component and pasted here. It was a long standing thing, if we are not using `<br />` it is not preserving the new lines when the tooltip content is rendered. 
https://github.com/webstudio-is/webstudio/blob/main/apps/builder/app/builder/features/style-panel/sections/filter/filter-content.tsx#L76-L86

Comment (kof) [reply apps/builder/app/builder/features/style-panel/sections/backdrop-filter/backdrop-filter.tsx:97]: what does it look like without those breaks?

Comment (JayaKrishnaNamburu) [reply apps/builder/app/builder/features/style-panel/sections/backdrop-filter/backdrop-filter.tsx:97]: with `<br />`
Screenshot 2024-05-07 at 14 10 04

without `<br />`
Screenshot 2024-05-07 at 14 09 53

**Comment by JayaKrishnaNamburu** [comment]
> * [ ]  just noticed - if you set backdrop-filter in the advanced panel, main panel will be set (blue title) but without the layers

Just noticed the reason for the same here, https://github.com/webstudio-is/webstudio/issues/3294#issuecomment-2097775421 let's discuss there on how to handle this. And i will add to this PR.

File: apps/builder/app/builder/features/style-panel/sections/backdrop-filter/backdrop-filter.tsx (hunk 1 above)
Comment (kof) [reply apps/builder/app/builder/features/style-panel/sections/backdrop-filter/backdrop-filter.tsx:97]: Ohh this should be handled by enforcing the maximum tooltip width and not with br
